python/private/bzlmod/pip.bzl

```
@@ -205,10 +207,14 @@ def _create_whl_repos(module_ctx, pip_attr, whl_map, whl_overrides):
             group_deps = group_deps,
         )
 
-        if whl_name not in whl_map[hub_name]:
-            whl_map[hub_name][whl_name] = {}
-
-        whl_map[hub_name][whl_name][_major_minor_version(pip_attr.python_version)] = pip_name + "_"
+        major_minor = _major_minor_version(pip_attr.python_version)
+        whl_map[hub_name].setdefault(whl_name, []).append(
+            whl_alias(
+                repo = repo_name,
+                version = major_minor,
+                config_setting = Label("//python/config_settings:is_python_" + major_minor),
```

I'm guessing the Label() call is to canonicalize within the @rules_python context because the actual usage is in the `@pip` context? Just drop a comment to mention that. This still trips me up and I helped someone today with the same thing.

```suggestion
                # Call Label() to canonicalize because its used in a different context
                config_setting = Label("//python/config_settings:is_python_" + major_minor),
```